common/prque/prque.go

```diff
@@ -27,24 +27,24 @@ type Prque struct {
 }
 
 // New creates a new priority queue.
-func New(setIndex SetIndexCallback) *Prque {
-	return &Prque{newSstack(setIndex)}
+func New(invert bool, setIndex SetIndexCallback) *Prque {
```

Comment:
If instead of `New(invert bool, ...) ` you turned it into two functions: `New` and `NewInverted`, then the original signature could remain, and the changes wouldn't touch as much code. Plus it's IMO a bit unelegant when the calling code which uses the 'default'  always have to pass in a `false` as the first argument